kentik_api_library/kentik_api/requests_payload/manual_mitigations_payload.py

```diff
@@ -0,0 +1,26 @@
+from kentik_api.public.manual_mitigation import ManualMitigation
+import json
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass()
+class CreateRequest:
```

Comment:
This duplication is no longer needed, after `methodId` field in `public.ManualMitigation` turned out to be `methodID`. In case of other payload classes duplicating classes in `public` i think we should import corresponding public class into payload file and delete duplicated code, but this is job for another PR